pkg/cluster/internal/providers/podman/provider.go

```diff
@@ -42,6 +42,7 @@ import (
 
 // NewProvider returns a new provider based on executing `podman ...`
 func NewProvider(logger log.Logger) provider.Provider {
+	logger.Warn("enabling experimental podman provider")
```

Comment:
in the future it might be nice to point to how to contact us along with this message